vc/examples-ext-v1.jsonld

```diff
@@ -135,13 +134,43 @@
     "CertifiedMillTestReport": {
       "@id": "ex:CertifiedMillTestReport",
       "@context": {
-       "@version": 1.1,
-       "@protected": true
+        "@version": 1.1,
+        "@protected": true
       }
     },
     "cmtr": {
-         "@id": "hetc:cmtr",
-         "@type": "@json"
+      "@id": "hetc:cmtr",
+      "@type": "@json"
+    },
+    "DIDConnection": {
+      "@id": "ex:DIDConnection",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "ex": "https://example.org/examples#",
+        "threadID": {
+          "@id": "ex:threadID",
+          "@type": "xsd:string"
+        },
+        "inviteeDID": {
+          "@id": "ex:inviteeDID",
+          "@type": "xsd:birthdate"
+        },
+        "inviterDID": {
+          "@id": "ex:inviterDID",
+          "@type": "xsd:string"
+        },
+        "inviterLabel": {
+          "@id": "ex:inviterLabel",
+          "@type": "xsd:string"
+        },
+        "state": {
```

Comment:
this is the connection record's state, right? I'm not sure if this field will remain in the end, but I suggest you rename to "connectionState" or something similar in the meanwhile

Reply:
already credential name is didconnection, so it is gonna stutter.

Reply:
done